### PR TITLE
Some fixes, update locale in files and update obconf-qt to 100% (Portuguese language)

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

File: compton-conf/compton-conf_pt.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="pt">
+<TS version="2.0" language="pt_PT">
```
Comment (palinek) [inline compton-conf/compton-conf_pt.ts]: Language should match the file name.
(repeats in other files also)

**Review by palinek — CHANGES_REQUESTED** [review]

**Comment by palinek** [comment]
Please, rebase to current master to resolve the conflicts...

**Comment by smarquespt** [comment]
@palinek 

I could do that, but I need some assistance. I´m not experienced in git usage and I´m afraid that anything I do could end up destroying (so to speak) my local clone and I will have to the work all over again.

SO my guess is that I will have to do:

1 - git fetch upstream master
2 - git merge upstream/master

Now the merge errors will appear. Try to fix them, if i can.

3 - git add file_name
4 - git commit file_name
5 - git push origin master
6 - git rebase HEAD~2

Is this correct?

**Comment by palinek** [comment]
There may be better ways (I'm also not a git guru), but I'd say something like

```
git fetch upstream
git rebase upstream/master
```

Now you'll get the conflicts... to see them:

```
git status
```

You should see something like:

```
rebase in progress; onto dd35624
You are currently rebasing branch 'mst_test' on 'dd35624'.
  (fix conflicts and then run "git rebase --continue")
  (use "git rebase --skip" to skip this patch)
  (use "git rebase --abort" to check out the original branch)

Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        modified:   compton-conf/compton-conf_pt.ts
        modified:   libfm-qt/libfm-qt_pt.ts
        modified:   liblxqt/liblxqt_pt.ts
        modified:   lximage-qt/lximage-qt_pt.ts
        modified:   lxqt-about/lxqt-about_pt.ts
        modified:   lxqt-admin/lxqt-admin-time/lxqt-admin-time_pt.ts
        modified:   lxqt-admin/lxqt-admin-user/lxqt-admin-user_pt.ts
        modified:   lxqt-config/liblxqt-config-cursor/lxqt-config-cursor_pt.ts
        modified:   lxqt-config/lxqt-config-appearance/lxqt-config-appearance_pt.ts
        modified:   lxqt-config/lxqt-config-file-associations/lxqt-config-file-associations_pt.ts
        modified:   lxqt-config/lxqt-config-input/lxqt-config-input_pt.ts
        modified:   lxqt-config/lxqt-config-monitor/lxqt-config-monitor_pt.ts
        modified:   lxqt-globalkeys/lxqt-config-globalkeyshortcuts/lxqt-config-globalkeyshortcuts_pt.ts
        modified:   lxqt-notificationd/lxqt-config-notificationd/lxqt-config-notificationd_pt.ts
        modified:   lxqt-openssh-askpass/lxqt-openssh-askpass_pt.ts
        modified:   lxqt-policykit/lxqt-policykit-agent/lxqt-policykit-agent_pt.ts
        modified:   lxqt-powermanagement/lxqt-config-powermanagement/lxqt-config-powermanagement_pt.ts
        modified:   lxqt-powermanagement/lxqt-powermanagement/lxqt-powermanagement_pt.ts
        modified:   lxqt-runner/lxqt-runner_pt.ts
        modified:   lxqt-session/lxqt-config-session/lxqt-config-session_pt.ts
        modified:   lxqt-session/lxqt-session/lxqt-session_pt.ts
        modified:   obconf-qt/obconf-qt_pt.ts
        modified:   qterminal/qterminal_pt.ts

Unmerged paths:
  (use "git reset HEAD <file>..." to unstage)
  (use "git add <file>..." to mark resolution)

        both modified:   lxqt-config/lxqt-config-brightness/lxqt-config-brightness_pt.ts
        both modified:   lxqt-config/lxqt-config-locale/lxqt-config-locale_pt.ts
        both modified:   lxqt-session/lxqt-leave/lxqt-leave_pt.ts
        both modified:   lxqt-sudo/lxqt-sudo_pt.ts

```

Fix the conflicting files (search for the conflict marks like `<<<<<<<`) and mark the files as resolved:

```
git add lxqt-config/lxqt-config-brightness/lxqt-config-brightness_pt.ts
...
```

When all files are resolved:

```
git rebase --continue
```

Now you should be successfully done with merging. You need to push it to remote (`origin` in your case):

```
git push --force
```

(note that you need the `--force` flag because you are "changing the history").

**Comment by smarquespt** [comment]
@palinek 
Happened waht I was afraid of. Closing

**Comment by palinek** [comment]
@smarquespt no problem. I've made it for you... see @d60ae4cd388b46abd9d69cc7c70aaad8b052da51

**Comment by smarquespt** [comment]
@palinek 
Thanks for all your work and assistance.

**Comment by paulolieuthier** [comment]
@smarquespt thank _you_! 👍

**Pull request closed without merging**